docs/rules/no-missing-import.md

```diff
@@ -46,6 +46,8 @@ import existingModule from "existing-module";
 
 #### allowModules
 
+This can also be configured as a shared setting [`settings.allowModules`](../shared-settings.md#allowmodules)
+
```

Comment:
mmm, so having the entire description in one place? 🤔

---

#### allowModules

This can be configured in the rule options or as a shared setting [`settings.allowModules`](../shared-settings.md#allowmodules).
Please see the shared settings documentation for more information.

Reply:
so, no need to repeat it in every rule doc, we can just maintain the shared settings docs.

Reply:
Okey, can do!

I will get to it first thing after breakfast :)